docs/concepts/bulk-update.md

```diff
@@ -211,3 +211,5 @@ There is no change in other APIs. All the other APIs supports only latest versio
 > Only one update operation can be performed at a time.
 
 > There is no way to delete only the latest version or revert back to original version.
+
+> We do not support updating PatientID field from a non-null to a null value.
```

Comment:
@esmadau, no we keep as it is since it is not part of DICOM standard, something we follow in other places like extended query tag. Even in azure docs, it is kept different.